ibis/pandas/aggcontext.py

```diff
@@ -331,7 +331,7 @@ def agg(self, grouped_data, function, *args, **kwargs):
         order_by = self.order_by
 
         # if we don't have a grouping key, just call into pandas
-        if not group_by:
+        if not group_by and not order_by:
```

Comment:
Why does `and not order_by` need to be added to the conditional? This new control flow logic does not seem to be related to what is being tested below.